### DISCUSSION TITLE
Any example using Redis

### DISCUSSION THREAD

**Discussion by ayoubAnbara** [MarcGiffing/bucket4j-spring-boot-starter#161] [Q&A] [answered]
I am looking for an example using Redis.
Thank you.

**Comment by MarcGiffing** [comment, marked as answer]
There are now examples:
https://github.com/MarcGiffing/bucket4j-spring-boot-starter/tree/master/examples
https://github.com/MarcGiffing/bucket4j-spring-boot-starter?tab=readme-ov-file#overview-cache-autoconfiguration
